src/ast/passes/semantic_analyser.cpp

```diff
@@ -431,6 +431,16 @@ void SemanticAnalyser::visit(Builtin &builtin)
   }
 }
 
+namespace {
+bool skip_key_validation(const Call &call)
+{
+  if (call.func == "print" || call.func == "clear" || call.func == "zero" ||
+      call.func == "len")
+    return true;
+  return false;
```

Comment:
style nit: maybe this instead
```suggestion
  return (call.func == "print" || call.func == "clear" || call.func == "zero" ||
      call.func == "len");
```